src/site/content/en/blog/schemeful-samesite/index.md

```diff
@@ -0,0 +1,378 @@
+`about:config` the console will display message for Schemeful Same-Site issues.
+You may see the following on your site:
+
+- "Cookie `cookie_name` **will be soon** treated as cross-site cookie against
+  `http://site.example/` because the scheme does not match."
+- "Cookie `cookie_name` **has been** treated as cross-site against
+  `http://site.example/` because the scheme does not match."
+
+## FAQ
+
+### My site is already fully available on HTTPS, why am I seeing issues in my browser's DevTools?
+
+It's possible that some of your links and subresources still point to insecure
+URLs.
+
+One way to fix this issue is to use [HTTP
+Strict-Transport-Security](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security) (HSTS)
+and the `includeSubDomain` directive. With HSTS + `includeSubDomain` even if one
+of your pages accidentally includes an insecure link the browser will
+automatically use the secure version instead.
+
+### What if I can't upgrade to HTTPS?
+
+While we strongly recommend that you upgrade your site entirely to HTTPS to
+protect your users, if you're unable to do so yourself we suggest speaking with
+your hosting provider to see if they can offer that option. If you self-host,
+then [Let's Encrypt](https://letsencrypt.org/) provides a number of tools to
+install and configure a certificate. You can also investigate moving your site
+behind a CDN or other proxy that can provide the HTTPS connection.
+
+If that's still not possible then try relaxing the `SameSite`
+protection on affected cookies.
+
+- In cases where only `SameSite=Strict` cookies are being blocked you can lower
+  the protection to `Lax`.
+- In cases where both `Strict` and `Lax` cookies are being blocked and your
+  cookies are being sent to (or set from) a secure URL you can lower the
+  protections to `None`.
+  - This workaround will **fail** if the URL you're sending cookies to (or
+    setting them from) is insecure. This is because `SameSite=None` requires the
+    `Secure` attribute on cookies which means those cookies may not be sent or
+    set over an insecure connection. In this case you will be unable to access
+    that cookie until your site is upgraded to HTTPS.
+  - Remember, this is only temporary as eventually third-party cookies will be
+    phased out entirely.
+
+### How does this affect my cookies if I haven't specified a `SameSite` attribute?
+
+Cookies without a `SameSite` attribute are treated as if they specified
+`SameSite=Lax` and the same cross-scheme behavior applies to these cookies as
+well. Note that the temporary exception to unsafe methods still applies, see
+[the Lax + POST mitigation in the Chromium `SameSite`
+FAQ](https://www.chromium.org/updates/same-site/faq) for more information.
+
+### How are WebSockets affected?
+
+Websocket connections will still be considered same-site if they're the same
+secureness as the page.
+
+For same-site:
+  - `wss://` ↔ `https://`
```

Comment:
Accessibility tip: Could you spell out the relationship represented with this character "↔"? It's listed as "left right arrow" in Unicode which is not very helpful for anyone using a screen reader. https://www.compart.com/en/unicode/U+2194

Reply:
*this is also the reason why I changed the check mark sign in the table to one that has a simpler name

Reply:
Replaced the character with explicit text.